### PR TITLE
add an index and go make some small tweaks

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@aluzzardi @vieux @fredlf

File: discovery/README.md [hunk 1]
```diff
@@ -138,8 +138,6 @@ $ swarm list zk://<zookeeper_addr1>,<zookeeper_addr2>/<path>
 ```bash
 # start the manager on any machine or your laptop
 $ swarm manage -H <swarm_ip:swarm_port> nodes://<node_ip1:2375>,<node_ip2:2375>
-#or
-$ swarm manage -H <swarm_ip:swarm_port> nodes://<node_ip1:2375>,<node_ip2:2375>
```
Comment (vieux) [inline discovery/README.md]: here it should be 
`$ swarm manage -H <swarm_ip:swarm_port> <node_ip1:2375>,<node_ip2:2375>`

Comment (SvenDowideit) [inline discovery/README.md]: @vieux what about the other examples of `swarm manage`?

Comment (vieux) [inline discovery/README.md]: @SvenDowideit no it's only for node

**Comment by SvenDowideit** [comment]
aaaand, I have another fix for this one mo.

File: discovery/README.md (hunk 1 above)
Comment (vieux) [inline discovery/README.md]: Please use 
```
$ swarm manage -H <swarm_ip:swarm_port> nodes://<node_ip1:2375>,<node_ip2:2375>
#or
$ swarm manage -H <swarm_ip:swarm_port> <node_ip1:2375>,<node_ip2:2375>
```

**Comment by vieux** [comment]
@SvenDowideit I added one more comment

**Comment by SvenDowideit** [comment]
@vieux @aluzzardi P'TAL

**Comment by fredlf** [comment]
LGTM

**Comment by vieux** [comment]
LGTM

**Pull request merged**